application/forms/PreferenceForm.php

```diff
@@ -185,7 +186,15 @@ public function createElements(array $formData)
                 false
             );
         }
-
+        $config = Config::app()->getSection('global');
+        if (!isset($config->config_resource)) {
+            $missingConfigResource =
+                Text::create($this->translate(
+                    'No Configuration Database selected.'
+                    . 'To establish a valid database connection set the Configuration Database field.'
+                ));
+            $this->warning($missingConfigResource, false);
```

Comment:
```suggestion
            $this->warning(
                $this->translate(
                    'No Configuration Database selected.'
                    . 'To establish a valid database connection set the Configuration Database field.'),
                false
            );
```
No need for `Text::create` and variable `$missingConfigResource`.